app/models/cart.rb

```diff
@@ -6,4 +6,10 @@ class Cart < ActiveRecord::Base
   def total
     items.inject(0) { |sum, item| sum += item.price }
   end
+
+  def checkout
+    user.total_spent = total
```

Comment:
Consider declaring an instance method in your user class called `update_total_spent`. The method would look something like this. 

```ruby
def update_total_spent(amount)
  self.total_spent += amount
end
```

That way if the user already has a value for their total_spent attribute you can update that value rather than override that existing value. 

Then in this method you would call that method 

`user.update_total_spent(total)`